openpype/hosts/traypublisher/api/plugin.py

```diff
@@ -301,7 +303,21 @@ def get_pre_create_attr_defs(self):
                 single_item=True,
                 label="Reviewable representations",
                 extensions_label="Single reviewable item"
-            )
+            ),
+            TextDef(
+                "input_colorspace",
```

Comment:
I exposed this so we don't rely solely on colorspace input rules and the user has control to choose what's the colorspace of the given sequence

Reply:
Oh nice! I wasn't aware of that PR, indeed that serves the same purpose!